app/gateway-oss/2.4.x/external-plugins.md

```diff
@@ -40,12 +40,12 @@ an hypothetical Python plugin server called `pypluginserver.py`):
 pluginserver_names = go,python
 
 pluginserver_go_socket = /usr/local/kong/go_pluginserver.sock
-pluginserver_go_start_cmd = go-pluginserver -kong-prefix /usr/local/kong/ -plugins-directory /usr/local/kong/go-plugins
-pluginserver_go_query_cmd = go-pluginserver -dump-all-plugins -plugins-directory /usr/local/kong/go-plugins
+pluginserver_go_start_cmd = /usr/local/bin/go-pluginserver -kong-prefix /usr/local/kong/ -plugins-directory /usr/local/kong/go-plugins
```

Comment:
I think the python plugin server indeed installed in /usr/local/bin if it's not installed with `-u`.
But i'm not sure about go, i think normally it's in GOPATH/bin. @anton-mesnyankin could you
describe how do you install the go-pluginserver?

Reply:
Looks like Javier confirmed this with https://github.com/Kong/docs.konghq.com/pull/2979.